### PR TITLE
Atomic testing list is slow on demo environment 

### PR DESCRIPTION
**Pull request by RomuDeuxfois**
### Proposed changes

* Add new indexes to speed up SQL Join

### Related issues

* https://github.com/OpenBAS-Platform/openbas/issues/1184

### CODE REVIEW

File: openbas-api/src/main/java/io/openbas/migration/V3_33_ADD_Index_inject.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+public class V3_33__Add_index extends BaseJavaMigration {
+
+    @Override
+    public void migrate(Context context) throws Exception {
+        Statement select = context.getConnection().createStatement();
+        select.execute(""
+            + "CREATE INDEX idx_inject_inject_injector_contract on injects (inject_injector_contract);"
+            + "CREATE INDEX idx_injector_contract_injector on injectors_contracts (injector_id);"
+            + "CREATE INDEX idx_injector_inject on injectors (injector_id);"
+            + ""
```
Comment (savacano28) [inline openbas-api/src/main/java/io/openbas/migration/V3_33_ADD_Index_inject.java:19]: The last one is already present (pk). I think we should generally add indexes on foreign keys.

Comment (RomuDeuxfois) [reply openbas-api/src/main/java/io/openbas/migration/V3_33_ADD_Index_inject.java:18]: Fix

**Comment by RomuDeuxfois** [comment]
Ready for a second round

File: openbas-api/src/main/java/io/openbas/migration/V3_33_ADD_Index_inject.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.springframework.stereotype.Component;
+
+import java.sql.Statement;
+
+@Component
+public class V3_33__Add_index extends BaseJavaMigration {
+
```
Comment (savacano28) [inline openbas-api/src/main/java/io/openbas/migration/V3_33_ADD_Index_inject.java:11]: I think the file and the class have to be named the same.

**Review by savacano28 — APPROVED** [review]

**Pull request merged**